migrations/versions/0394_notification_event_log.py

```diff
@@ -0,0 +1,38 @@
+"""
+
+Revision ID: 0394_notification_event_log
+Revises: 0393_add_go_live_template
+Create Date: 2023-01-04 14:14:19.133958
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "0394_notification_event_log"
+down_revision = "0393_add_go_live_template"
+
+
+def upgrade():
+    op.create_table(
+        "notification_event_log",
+        sa.Column("id", sa.BigInteger(), nullable=False),
+        sa.Column("notification_id", postgresql.UUID(as_uuid=True), nullable=True),
+        sa.Column("created_at", sa.DateTime(), nullable=True),
```

Comment:
I know we always call it `created_at` but I do wonder if we should instead have something like `event_datetime` instead to make it less confusing (but maybe not worth the divergence). One to ponder and seek other opinions if you were to consider

Reply:
That's a good shout 👍

Reply:
Maybe. I think the bigger decision is whether to diverge from consistency or not, for the name itself any of those seem like sensible options

Reply:
It's also definitely something we _could_ revisit at any point fairly easily so I'm not considering it a blocker on this PR for now.

Reply:
i like the `_at` suffix for timestamps  and im fine with diverging from consistency considering that we may well be trying to match this timestamp with a timestamp given in an arbitrary field on the notification, so leaving it as created_at might be confusing. i like happened_at as a starter for ten